README.md

```diff
@@ -41,3 +41,12 @@ Ibis currently provides tools for interacting with the following systems:
 - [Spark](https://spark.apache.org) (Experimental)
 
 Learn more about using the library at http://docs.ibis-project.org.
+
+
+## Notes
+
+- OmniSciDB backend support is tested against the development release
```

Comment:
```suggestion
- OmniSciDB backend support is tested against a development release
```

Reply:
thanks @saulshanabrook I will change that.

Reply:
done!